test/sql/test_semi/T/test_any_value

```diff
@@ -0,0 +1,32 @@
+-- name: test_prune @system
+CREATE TABLE `sc3` (
+  `v1` bigint(20) NULL COMMENT "",
+  `a1` ARRAY<INT> NULL,
+  `a2` ARRAY<STRUCT<a INT, b INT>> NULL,
+  `m1` MAP<INT, INT> NULL,
+  `m2` MAP<INT, STRUCT<c INT, b ARRAY<INT>>> NULL,
+  `s1` STRUCT<s1 int, s2 ARRAY<STRUCT<a int, b int>>, s3 MAP<INT, INT>, s4 Struct<e INT, f INT>>
+) ENGINE=OLAP
+DUPLICATE KEY(`v1`)
+COMMENT "OLAP"
+DISTRIBUTED BY HASH(`v1`) BUCKETS 3
+PROPERTIES (
+"replication_num" = "1",
+"in_memory" = "false",
+"enable_persistent_index" = "false",
+"replicated_storage" = "true",
+"compression" = "LZ4"
+);
+
+insert into sc3 values (1, [1,2,3],[row(1,11),row(2,21),row(3,31)], map{1:11, 2:21}, map{1:row(11, [111, 221, 331]), 2:row(22, [221, 221, 331])}, row(1, [row(1,1), row(2,1)], map{1:11, 2:21}, row(1,1)));
+insert into sc3 values (2, [2,2,3],[row(1,12),row(2,22),row(3,32)], map{1:12, 2:22}, map{1:row(12, [112, 222, 332]), 2:row(22, [222, 222, 332])}, row(1, [row(1,2), row(2,2)], map{1:12, 2:22}, row(1,2)));
+insert into sc3 values (3, [3,2,3],[row(1,13),row(2,23),row(3,33)], map{1:13, 2:23}, map{1:row(13, [113, 223, 333]), 2:row(22, [223, 223, 333])}, row(1, [row(1,3), row(2,3)], map{1:13, 2:23}, row(1,3)));
+insert into sc3 values (1, [4,2,3],[row(1,14),row(2,24),row(3,34)], map{1:14, 2:24}, map{1:row(14, [114, 224, 334]), 2:row(22, [224, 224, 334])}, row(1, [row(1,4), row(2,4)], map{1:14, 2:24}, row(1,4)));
+insert into sc3 values (2, [5,2,3],[row(1,15),row(2,25),row(3,35)], map{1:15, 2:25}, map{1:row(15, [115, 225, 335]), 2:row(22, [225, 225, 335])}, row(1, [row(1,5), row(2,5)], map{1:15, 2:25}, row(1,5)));
+insert into sc3 values (3, [6,2,3],[row(1,16),row(2,26),row(3,36)], map{1:16, 2:26}, map{1:row(16, [116, 226, 336]), 2:row(22, [226, 226, 336])}, row(1, [row(1,6), row(2,6)], map{1:16, 2:26}, row(1,6)));
+insert into sc3 values (1, [7,2,3],[row(1,17),row(2,27),row(3,37)], map{1:17, 2:27}, map{1:row(17, [117, 227, 337]), 2:row(22, [227, 227, 337])}, row(1, [row(1,7), row(2,7)], map{1:17, 2:27}, row(1,7)));
+insert into sc3 values (2, [8,2,3],[row(1,18),row(2,28),row(3,38)], map{1:18, 2:28}, map{1:row(18, [118, 228, 338]), 2:row(22, [228, 228, 338])}, row(1, [row(1,8), row(2,8)], map{1:18, 2:28}, row(1,8)));
+insert into sc3 values (3, [9,2,3],[row(1,19),row(2,29),row(3,39)], map{1:19, 2:29}, map{1:row(19, [119, 229, 339]), 2:row(22, [229, 229, 339])}, row(1, [row(1,9), row(2,9)], map{1:19, 2:29}, row(1,9)));
+
+select any_value(v1), any_value(a1), any_value(a2), any_value(m1), any_value(m2), any_value(s1) from sc3;
+select any_value(a1), any_value(a2), any_value(m1), any_value(m2), any_value(s1) from sc3 where v1 = 2 group by v1;
```

Comment:
add more test cases?
1. for window functions
2. any_value(null)?
3. any_value(const value)?
4. any_value(distinct)?
5. other abnormal cases?

Reply:
1. doesn't support window functions
2. exists test case

Reply:
these unsupported cases to see the error report, at least do not crash.

Reply:
FE was forbidden it